client/src/views/client-area/BillingDetailsPage.vue

```diff
@@ -0,0 +1,20 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
```

Comment:
I think there already is a `BillingPage`, how is this one different? Should the other one be deleted?

Reply:
it is this page, simply renamed